index.html

```diff
@@ -388,4 +333,5 @@ <h2 class="main_title fw_bold white">Contact me</h2>
     <!-- end form section -->
   </div>
 </body>
+
```

Comment:
- [ ] There are a lot of comments in your index.html, kindly remove them and have this file only contain HTML code. Separating your blocks of code with empty lines and giving your section's names by their defined classes is already clear to know where a section is located in your code.